data/name-corrections/name-corrections.csv

```diff
@@ -0,0 +1,105 @@
+path,correct
```

Comment:
have we incorporated newest name corrections confirmed with Mo in winter? (noted somewhere in our notes document)

Reply:
they're been added in commit aa6c573.

conversation resolved